doc/release/2.11.0.rst

```diff
@@ -0,0 +1,586 @@
+Tarantool 2.11.0
+================
+
+Released on 2023-05-22.
+
+*   Release: :tarantool-release:`2.11.0`.
+*   Tag: ``2.11.0``.
+
+
+The 2.11 release of Tarantool includes many new features and fixes for Enterprise and Community versions. 2.11 is the long-term support (LTS) release with four years of maintenance. This means that you will continue to receive all the necessary security fixes and bug fixes throughout this period. You can learn more about the Tarantool release policy from the :doc:`corresponding document </release/policy>`.
+
+
+
+.. _enterprise:
+
+Enterprise
+----------
+
+
+.. _security_enhancements:
+
+Security enhancements
+~~~~~~~~~~~~~~~~~~~~~
+
+
+.. _encrypted_ssl_tls_keys:
+
+Encrypted SSL/TLS keys
+**********************
+
+Tarantool Enterprise now supports `encrypted <https://www.tarantool.io/en/enterprise_doc/security/#traffic-encryption>`_ SSL/TLS private key files protected with a password.
```

Comment:
It's better to create an intersphinx mapping to avoid absolute links to the Enterprise doc. Although this may be out of this PR's scope.